zqd/ingest/archivepcap.go

```diff
@@ -101,13 +102,15 @@ func (p *archivePcapOp) run(ctx context.Context) error {
 	}
 	combiner := zbuf.NewCombiner(zreaders, zbuf.RecordCompare(p.store.NativeOrder()))
 	defer combiner.Close()
-	// track stats on records produced from analyzers.
-	p.recordCounter.reader = combiner
 
-	if err := p.store.Write(ctx, p.zctx, p.recordCounter); err != nil {
+	if err := zbuf.CopyWithContext(ctx, p.writer, combiner); err != nil {
```

Comment:
There should be a p.writer.Close if this errors.

Reply:
right you are